docs/docs/contributing/testing.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: doc-page
+title: Testing in Dotty
+---
+
+<aside class="warning">
+This page should be updated as soon as scala-partest is removed
+</aside>
+
+Running all tests in Dotty is as simple as:
+
+```bash
+$ sbt
+> test
+```
+
+There are currently several forms of tests in Dotty. These can be split into
+two categories:
+
+## Unit tests
+These tests can be found in `<sub-project>/test` and are used to check
+functionality of specific parts of the codebase in isolation, such e.g.
+parsing, scanning and message errors.
+
+Running a single unit test class is as simple as:
+
+```bash
+$ sbt
+> testOnly absolute.path.to.TestClass
+```
+
+To filter out individual tests you can use:
+
+```bash
+$ sbt
+> testOnly absolute.path.to.TestClass -- *methodName
+```
+
+## Integration tests
+These tests are source files compiled by Dotty with an expected amount of
```

Comment:
These tests are Scala source files expected to compile with Dotty (pos tests), along with their expected output (run tests) or errors (neg tests).